loading.bs

```diff
@@ -381,6 +391,38 @@ for="certificate">extensions</dfn> map ([=Certificate Extensions=]) from OIDs to
 A <dfn>certificate chain</dfn> is a [=list=] of [=augmented certificates=], of
 which the first [=list/item=] is the <dfn for="certificate chain">leaf</dfn>.
 
+<h3 dfn-type=dfn export>Signed Exchange report</h3>
+
+A signed exchange report is a [=struct=] with the following items:
+
+<dl dfn-for="signed exchange report">
+: <dfn>result</dfn>
+:: The result string of loading signed exchange. This must be one of an empty
+    string, "<dfn>`ok`</dfn>", "<dfn>`mi_error`</dfn>",
+    "<dfn>`non_secure_origin`</dfn>", "<dfn>`parse_error`</dfn>",
+    "<dfn>`invalid_integrity_header`</dfn>",
+    "<dfn>`signature_verification_error`</dfn>",
+    "<dfn>`cert_verification_error`</dfn>", "<dfn>`cert_fetch_error`</dfn>",
+    "<dfn>`cert_parse_error`</dfn>",
+: <dfn>outer request</dfn>
+:: The [=request=] which the user agent sent to the server to load the signed exchange.
+: <dfn>outer response</dfn>
+:: The [=response=] which the user agent received from the server.
+: <dfn>inner URL</dfn>
+:: The logical [=URL=] of the signed exchange, if available. Otherwise, an empty
+    string.
+: <dfn>cert URL list</dfn>
+:: The list of [=URL=] of the first "`cert-url`" parameter of signed exchange,
```

Comment:
That's not quite right: the first "cert-url" parameter only has 1 URL, but there may be multiple signature headers, each with its own "cert-url".

Do we need to associate each certificate with its server IP, or is just having an unmatched bag ok?